.github/workflows/add_issues_projects.yaml

```diff
@@ -0,0 +1,23 @@
+name: "Adiciona Issues em Projects GitHub"
+
+on:
+  workflow_call:
+    inputs:
+      project_url:
+        description: "URL do projetcs github"
+        required: true
+        type: string
+      github_token:
+        description: "Personal token para leitura/escrita projects github"
+        required: true
+        type: string
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
```

Comment:
Seria legal ter uma versão fixada aqui para evitar supresas. Ainda mais porque essa action está em beta, então deve mudar de forma não-suportada com certa frequencia.

Reply:
Atualmente o projeto está na versão v0.0.4, eu tentei utilizar a tag v0 o pipe quebrou. 
Acho que podemos deixar na main até o projeto evoluir um pouco mais.

Reply:
Eu concordo, vai mudar muito e não vale a pena mesmo.

Reply:
Eu acho que vale a pena deixar fixo _justamente_ porque vai mudar muito :sweat_smile: 

Como é uma action que ainda está em desenvolvimento usar `main` representa um risco da pipeline quebrar porque a action mudou de forma inesperada. O próprio repo da action recomenda usar uma versão fixa:

> :rotating_light: This action is in beta, however the API is stable. Some breaking changes might occur between versions, but it is not likely to break as long as you use a specific SHA or version number :rotating_light: 

https://github.com/actions/add-to-project#current-status

Reply:
Obrigado @gomex e @lgfa29 pelas obsservações. 
Eu fiz o ajuste setando a versão fixa do provejo `v0.0.4` de acordo com a nossa necessidade vamos alterando a versão da actions.